code/mob/living/critter/bloodling.dm

```diff
@@ -0,0 +1,121 @@
+/mob/living/critter/bloodling
+	name = "bloodling"
+	desc = "A force of pure sorrow and evil. They shy away from that which is holy."
+	icon_state = "bling"
+
+	density = 1
+	hand_count = 1
+
+	ai_type = /datum/aiHolder/bloodling
+	is_npc = TRUE
+
+	can_burn = FALSE
+	canbegrabbed = FALSE
+	throws_can_hit_me = FALSE
+	reagent_capacity = 0
+	can_bleed = FALSE // has custom bleed effects already
+	metabolizes = FALSE
+	use_stamina = FALSE
+
+	New()
+		UpdateParticles(new/particles/bloody_aura, "bloodaura")
+
+		remove_lifeprocess(/datum/lifeprocess/blood)
+		remove_lifeprocess(/datum/lifeprocess/chems)
+		remove_lifeprocess(/datum/lifeprocess/fire)
+		remove_lifeprocess(/datum/lifeprocess/mutations)
+		remove_lifeprocess(/datum/lifeprocess/organs)
+		remove_lifeprocess(/datum/lifeprocess/skin)
+		remove_lifeprocess(/datum/lifeprocess/stuns_lying)
+		remove_lifeprocess(/datum/lifeprocess/viruses)
+		remove_lifeprocess(/datum/lifeprocess/blindness)
+		remove_lifeprocess(/datum/lifeprocess/radiation)
+
+		APPLY_ATOM_PROPERTY(src, PROP_MOB_RADPROT_INT, src, 100)
+		APPLY_ATOM_PROPERTY(src, PROP_MOB_RADPROT_EXT, src, 100)
+		APPLY_ATOM_PROPERTY(src, PROP_ATOM_FLOATING, src)
+		APPLY_ATOM_PROPERTY(src, PROP_MOB_NIGHTVISION, src)
+		..()
+
+	setup_hands()
+		..()
+		var/datum/handHolder/HH = hands[1]
+		HH.limb = new /datum/limb/bloodling_suck
+		HH.name = "blood suck"
+		HH.limb_name = HH.name
+		HH.can_hold_items = FALSE
+
+	setup_healths()
+		var/datum/healthHolder/brute = src.add_health_holder(/datum/healthHolder/bloodling_brute)
+		brute.value = 25
+		brute.maximum_value = 25
+		brute.last_value = 25
+		brute.damage_multiplier = 0.5
+
+		var/datum/healthHolder/burn = src.add_health_holder(/datum/healthHolder/bloodling_burn)
+		burn.value = 25
+		burn.maximum_value = 25
+		burn.last_value = 25
```

Comment:
any reason not to use add_hh_flesh and add_hh_flesh_burn here?

Reply:
It has stuff about chems which I don't think bloodling would use really

Reply:
they are still used for critters that don't metabolise